### ISSUE TITLE
[FEATURE REQUEST] Focus Mode Sharing between macOS and Android

### ISSUE THREAD

**Issue by sannidhyaroy** [agungrbudiman/Soduto#4]
Hey, I thought it would be interesting if adding such a feature within Soduto would be possible so that when my Mac turns on Focus Mode, my android device would also enable it. Google released Focus Mode back in Android 10 and Apple released it in macOS Monterey. Apple has released the Focus API, although I couldn't find the same for android, hence I know this is a very unlikely and odd feature request and probably not possible to implement, so I'm not expecting to take this seriously, but I just though of sharing it, if in case anyone thinks it's possible and feasible. Sorry in advance, if I'm making a selfish request!

**Comment by jsonMartin** [comment]
This is a great idea, I'd love this feature

**Comment by sannidhyaroy** [comment]
> This is a great idea, I'd love this feature

@jsonMartin I agree but this would kinda be tough to implement. I have written more about [this here](https://github.com/sannidhyaroy/Soduto/issues/5#issuecomment-1374964378)
